SPM Sample/SPM-sample-app.xcodeproj/project.pbxproj

```diff
@@ -525,8 +525,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/ContentSquare/CS_iOS_SDK.git";
 			requirement = {
-				kind = exactVersion;
-				version = 4.25.2;
+				kind = upToNextMajorVersion;
+				minimumVersion = 4.25.2;
```

Comment:
I personally prefer having "exact" for two reasons:

- it leaves some git history that the version was updated and therefore the sample app tested
- it removes the need to discuss it every time we change it